src/conduit/core.cljs

```diff
@@ -43,7 +45,12 @@
 ;; initialize controllers
 (defonce init-ctrl (citrus/broadcast-sync! reconciler :init))
 
-(router/start! #(citrus/dispatch! reconciler :router :push %) routes)
+(router/start! (fn [route]
```

Comment:
What is this code trying to achieve?

Reply:
It checks if the current route is `:logout` then it will dispatch the `:user :logout` action. I know it's bad to hardcode it here, but I don't know of a better way to handle it for `:logout`.